ghmirror/data_structures/redis_data_structures.py

```diff
@@ -67,10 +68,19 @@ def _scan_iter(self):
         """
         cursor = '0'
         while cursor != 0:
-            cursor, data = self.cache.scan(cursor)
+            cursor, data = self.wr_cache.scan(cursor)
             for item in data:
                 yield self._deserialize(item)
 
+    @staticmethod
+    def _get_connection(host='localhost'):
```

Comment:
There's an issue here...host will never use the default

Reply:
you're right. good catch. When I was testing locally I didn't set the environment variables for primary or reader and so this essentially set host = None instead of to localhost but that was still interpreted by Redis to be localhost.

It is now improved. if `PRIMARY_ENDPOINT` is not set it will default to localhost, and the `READER_ENDPOINT` will default to the same.